test/cases/grid-options.no-autoplace.out.css

```diff
@@ -153,7 +153,7 @@
 .grid-autoplace {
   display: -ms-grid;
   display: grid;
-  -ms-grid-columns: 1fr 30px 1fr 30px 1fr;
+  -ms-grid-columns: 1fr 1fr 1fr;
```

Comment:
This line should still be adding the grid gap to the columns.

Autoplacement is enabled in this test so grid-gap is applied.

Reply:
Autpalcement us not enabled in this test. It's enabled in `grid-options.autoplace.out.css`

Reply:
We just process `grid-options.css` which contains grid area and autoplacement algorithms. For each option (with and without autoplacement) we expect different outputs (`grid-options.autoplacement.css` and `grid-options.no-autoplacement.css`).

Reply:
Oh right, yeah sorry. Ignore me 😅